lib/es.rb

```diff
@@ -3,23 +3,41 @@
 module ES
   INDEX_PREFIX = "#{Rails.env}-usasearch".freeze
 
-  def self.client_reader
-    @@client_reader ||= Elasticsearch::Client.new(log: Rails.env == 'development', host: reader_host)
```

Comment:
I was concerned that the change from `@@client_reader` to `@client_reader` would cause these clients to be re-created much more frequently than they were before (and therefore new ES connections to be created much more frequently than before). But I added some logging on demo3 that would show me whenever these new clients are created. I saw that connections were still being re-used, so I'm no longer worried.

In retrospect this makes sense because these are actual module instance variables and not object instance variables, so they ultimately belong to the classes this module gets mixed into, not instances. Once a particular class creates its `@client_reader`, it holds onto it for the rest of its life 👍